drivers/usb/device/usb_dc_stm32.c

```diff
@@ -173,9 +181,15 @@ static int usb_dc_stm32_clock_enable(void)
 	struct device *clk = device_get_binding(STM32_CLOCK_CONTROL_NAME);
 	struct stm32_pclken pclken = {
 
+#ifdef CONFIG_USB_HS_BASE_ADDRESS
+		.bus = STM32_CLOCK_BUS_AHB1,
+		.enr = LL_AHB1_GRP1_PERIPH_OTGHS
+#else /* CONFIG_USB_HS_BASE_ADDRESS */
+
 #ifdef USB
```

Comment:
I think we need to change USB macro to something more readable.

Reply:
```
/*
 * USB and USB_OTG_FS are defined in STM32Cube HAL and allows to distinguish
 * between two kind of USB DC. STM32 F0, F3, and L0 series support USB device
 * controller. STM32 F4 and F7 series support USB_OTG_FS device controller.
 * STM32 F1 and L4 series support either USB or USB_OTG_FS device controller.
 *
 * WARNING: Don't mix USB defined in STM32Cube HAL and CONFIG_USB from Zephyr
 * Kconfig system.
 */
```
I know that USB macro is really confusing, that's why I have already add the above comment inside the driver.